README.md

```diff
@@ -101,3 +101,8 @@ spec:
 ```
 
 That is it! Now that you have the secondary CNIs and other related plugins deployed, you may need to prep the nodes before you can actually create Multus Networks and assign them to Pods. In order to do so, use Luigi's own HostPlumber plugin: https://github.com/platform9/luigi/blob/master/hostplumber/README.md
+
+
+##### Dev note
+This project need to migrate to Kubebuilder/v4.
```

Comment:
Is moving to v4 still required?

Reply:
v3 is deprecated 2 years ago, weebhook that we added do not scaffold automatically.
its in patched working state now. 
ideal to migrate
https://kubebuilder.io/migration/v3vsv4.html